### PR TITLE
PauseGame SaveGame and LoadGame

### PR DESCRIPTION
**Pull request by swaggyavi**
Pause is added using ESC, can save most data and load most data

### CODE REVIEW

**Comment by dallinson** [comment]
Uhh, why are the CI files deleted?

File: core/src/main/java/com/mygdx/auber/entities/Prisoners.java [hunk 1]
```diff
@@ -62,4 +72,35 @@ public static void render(Batch batch) {
...
+
+        for (Sprite sprite : Prisoners.prisoners) {
+            prisonerLocations.add(new Vector2(sprite.getX(), sprite.getY()));
+        }
+        return prisonerLocations.toString();
+    }
+
+    // TODO: Needs testing.
+    public static void LoadFromString(String value1) {
+        String value = "[(1936.0,3632.0), (1968.0,3632.0), (2032.0,3632.0), (2064.0,3632.0), (2128.0,3632.0), (2160.0,3632.0)]";
```
Comment (dallinson) [inline core/src/main/java/com/mygdx/auber/entities/Prisoners.java:90]: Is this meant to be JSON or similar?  Could try using a JSON library if so.

Comment (swaggyavi) [reply core/src/main/java/com/mygdx/auber/entities/Prisoners.java:90]: Prisoners isn't currently finished right now - it literally does nothing, i'm gonna fix that tomorrow.

Comment (tmh528) [reply core/src/main/java/com/mygdx/auber/entities/Prisoners.java:90]: There are no prisoners visible for anyone at the moment, there hasn't been for a while so you're good.

**Pull request merged**